mono/mini/interp/interp.c

```diff
@@ -1384,11 +1384,7 @@ static InterpMethodArguments* build_args_from_sig (MonoMethodSignature *sig, Int
 #if DEBUG_INTERP
 			g_print ("build_args_from_sig: margs->fargs [%d]: %p (%f) (frame @ %d)\n", int_f, margs->fargs [int_f], margs->fargs [int_f], i);
 #endif
-#if SIZEOF_VOID_P == 4
-			int_f += 2;
-#else
-			int_f++;
-#endif
+			int_f ++;
```

Comment:
We are still allocating 2 slots for R8 in the initial loop. Is this just hiding the real issue ?

Reply:
For r4, we only allocate 1 slot.

Reply:
Shouldn't we always allocate only 1 slot for both r4 and r8 (flen from above)

Reply:
Yes